### PR TITLE
touchdetector: filter out pedantic stuff from the GCC

### PR DESCRIPTION
**Pull request by matz-e**

### CODE REVIEW

**Review by olupton — APPROVED** [review]
LGTM, I assume you tested this locally with `touchdetector@develop`...

File: bluebrain/repo-bluebrain/packages/touchdetector/package.py [hunk 1]
```diff
@@ -71,6 +71,12 @@ def patch(self):
             filter_file(r'(int messageLength) = -1;$',
                         r'\1 = 0;',
                         'touchdetector/DistributedTouchDetector.cxx')
+        elif self.spec.satisfies('@develop'):
+            filter_file(
+                r'-Werror',
+                '-Werror -Wno-error=stringop-overflow',
```
Comment (pramodk) [inline bluebrain/repo-bluebrain/packages/touchdetector/package.py:77]: Bob effect!

Comment (matz-e) [reply bluebrain/repo-bluebrain/packages/touchdetector/package.py:77]: No bob there, just a newer compiler with more automatic flags :)

Comment (matz-e) [reply bluebrain/repo-bluebrain/packages/touchdetector/package.py:77]: Haven't had time to fix that in the code itself, need to get back to that.

**Pull request merged**